packages/xod-client/src/messages/actions.js

```diff
@@ -1,21 +1,17 @@
+import * as R from 'ramda';
 import * as ActionType from './actionTypes';
 import { MESSAGE_TYPE } from './constants';
 import { STATUS } from '../utils/constants';
 
 const getTimestamp = () => new Date().getTime();
 
-export const addMessage = (
-  type,
-  messageData,
-  persistent = false,
-  id = null
-) => ({
+export const addMessage = (type, messageData, id = null) => ({
   type: ActionType.MESSAGE_ADD,
   payload: messageData,
   meta: {
     id,
     type,
-    persistent,
+    persistent: R.propOr(true, 'persistent', messageData),
```

Comment:
Could it default to `true` only if `type` is `MESSAGE_TYPE.ERROR`?

Reply:
I thought about it and even did it at first. But when I began to place persistent flags in message objects, I realized that there is confusion. So I decided to do it persistent by default for all types of messages.